pkg/tbtc/registry_test.go

```diff
@@ -223,6 +224,10 @@ func (mph *mockPersistenceHandle) Archive(directory string) error {
 	panic("not implemented")
 }
 
+func (mph *mockPersistenceHandle) Delete(directory string, name string) error {
```

Comment:
Do we need it? The `registry` uses `persistence.ProtectedHandle` that doesn't expose the `Delete` function.

Reply:
fa6ee0abbc04f580fb10431403ceb5058b87a869